toolbackup/pom.xml

```diff
@@ -170,12 +170,14 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
+            <version>1.11.83</version>
```

Comment:
Can use a property for these two to hint to a future developer that either these two are upgraded at once or more likely, deleted at the same time